notebooks/text-to-image-genai/text-to-image-genai.ipynb

```diff
@@ -0,0 +1,393 @@
+{
```

Comment:
<p>&gt;&gt;Run inference using the standard <a href="https://openvino-doc.iotg.sclab.intel.com/seba-test-4/learn-openvino/llm_inference_guide/genai-guide/genai-use-cases.html#using-genai-for-text-to-image-generation" target="_blank">Text to Image pipeline</a> from OpenVINO GenAI.</p><p> </p><p>what you mean under "standard", is there something non-standard?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2472/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Removed